test/cluster/functionality.sh

```diff
@@ -73,7 +73,10 @@ for elasticsearch_pod in $( oc get pods --selector component="${OAL_ELASTICSEARC
 	os::cmd::expect_success_and_text "oc get pod ${elasticsearch_pod} -o jsonpath='{ .status.containerStatuses[?(@.name==\"elasticsearch\")].ready }'" "true"
 
 	os::log::info "Checking that Elasticsearch pod ${elasticsearch_pod} recovered its indices after starting..."
-	if oc logs "${elasticsearch_pod}" | grep -E "\[cluster\.service\s*\]" | grep -q "new_master"; then
+	os::cmd::try_until_text "curl_es '${elasticsearch_pod}' '/_cluster/state/master_node' -w '%{response_code}'" "}200$" "$(( 10*TIME_MIN ))"
+	es_master_id="$( curl_es "${elasticsearch_pod}" "/_cluster/state/master_node" | jq --raw-output '.master_node' )"
```

Comment:
> `jq` generally available on CentOS and RHEL, or are special repos required?

Yes, it's available on RHEL and CentOS just fine. It looks like it has just one dependency -- `oniguruma`. 

<details>
<summary> Here is an install log from a RHEL machine </summary>

```
Loaded plugins: amazon-id, rhui-lb, search-disabled-repos
Resolving Dependencies
--> Running transaction check
---> Package jq.x86_64 0:1.5-1.el7 will be installed
--> Processing Dependency: libonig.so.2()(64bit) for package: jq-1.5-1.el7.x86_64
--> Running transaction check
---> Package oniguruma.x86_64 0:5.9.5-3.el7 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

================================================================================
 Package    Arch    Version      Repository                                Size
================================================================================
Installing:
 jq         x86_64  1.5-1.el7    centos-paas-sig-openshift-origin15-rpms  152 k
Installing for dependencies:
 oniguruma  x86_64  5.9.5-3.el7  centos-paas-sig-openshift-origin15-rpms  127 k

Transaction Summary
================================================================================
Install  1 Package (+1 Dependent package)

Total download size: 280 k
Installed size: 906 k
Downloading packages:
--------------------------------------------------------------------------------
Total                                              331 kB/s | 280 kB  00:00     
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Installing : oniguruma-5.9.5-3.el7.x86_64                                 1/2 
  Installing : jq-1.5-1.el7.x86_64                                          2/2 
  Verifying  : oniguruma-5.9.5-3.el7.x86_64                                 1/2 
  Verifying  : jq-1.5-1.el7.x86_64                                          2/2 

Installed:
  jq.x86_64 0:1.5-1.el7                                                         

Dependency Installed:
  oniguruma.x86_64 0:5.9.5-3.el7                                                

Complete!
```

</details>